core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# BAREOS - Backup Archiving REcovery Open Sourced
+#
+# Copyright (C) 2023-2023 Bareos GmbH & Co. KG
+#
+# This program is Free Software; you can redistribute it and/or
+# modify it under the terms of version three of the GNU Affero General Public
+# License as published by the Free Software Foundation, which is
+# listed in the file LICENSE.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.
+#
+# Author: Bareos Team
+#
+"""bareos-fd-postgresql is a plugin to backup PostgreSQL cluster
+using BareosFdPluginPostgreSQL
+"""
+from sys import version_info
+
+# Provided by the Bareos FD Python plugin interface
+import bareosfd
+from bareosfd import *
+
+# This module contains the wrapper functions called by the Bareos-FD, the
+# functions call the corresponding methods from your plugin class
+import BareosFdWrapper
+
+# from BareosFdWrapper import parse_plugin_definition, handle_plugin_event,
+# start_backup_file, end_backup_file, start_restore_file, end_restore_file,
+# restore_object_data, plugin_io, create_file, check_file, handle_backup_file
+# # noqa
+
+from BareosFdWrapper import *  # noqa
+
+
+def load_bareos_plugin(plugindef):
+    """
+    This function is called by the Bareos-FD to load the plugin
+    We use it to instantiate the plugin class
+    """
+    if version_info.major >= 3 and version_info.minor < 6:
```

Comment:
This does not make sense: f-strings have been introduced in python 3.6, so using them in case python is older than 3.6 will for sure not work.

Reply:
In that particular case, you make a point, but who knows when we will have to limit this to something higher like 3.11 in a few?
Our concern here is Jenkin in EL7 brp-python-bytecompress using python2 when it should use only python3.
It is still easy here to downgrade the code to make it failing gracefully in something lower than 3.6

Reply:
Jenkins for EL7 has been fixed, by requiring to use only python3